code/renderers/vue3/src/mount.ts

```diff
@@ -0,0 +1,13 @@
+import { type StoryContext, type VueRenderer } from './public-types';
+import { h } from 'vue';
+import { type BaseAnnotations } from '@storybook/types';
+
+export const mount: BaseAnnotations<VueRenderer>['mount'] = (context: StoryContext) => {
+  return async (Component, options) => {
+    if (Component) {
+      context.originalStoryFn = () => () => h(Component, options?.props, options?.slots);
```

Comment:
Does this work correctly with reactivity?

Reply:
We actually don't support reactivity in mount. It will always do a force remount. As the play function is not called with reactive rerenders.